packages/legend-query/src/stores/QueryBuilderExplorerState.ts

```diff
@@ -337,6 +337,24 @@ export const getPropertyNodeMappingData = (
   return { mapped: false, skipMappingCheck: false };
 };
 
+export const getSubTypeNodeMappingData = (
+  subclass: Class,
+  parentMappingData: QueryBuilderPropertyMappingData,
+  mapping: Mapping | undefined,
+): QueryBuilderPropertyMappingData => {
+  const rootSetImpl = mapping
+    ? getRootSetImplementation(mapping, subclass)
```

Comment:
I believe we don't just find the root set implementation for the subclass but all of the subclass of that subclasses.

Reply:
@akphi what do you mean by this comment?

Reply:
Rethinking about this and this algo seems to be correct. I think the handling for the `extends` is separate after thinking about it a bit more. `extends` is when you look at stuff mapped to the supertype.
So here we are going with only showing subclasses if they are mapped.

Reply:
Let's check with @pierredebelen or John Pentelow to see how we should implement this. Let say we have 


```
Class Person {
  name: String[1];
}

Class Teacher extends Person {
  school: String[1];
}

Class Student extends Person {
  class: String[1];
}
```

I see 2 potential approach to determine if property `name` of `Person` is mapped:

1. Find root class mapping for `Person`, if there is none, `name` is considered not mapped
2. In any of the subclasses of `Person` is mapped, if `name` is mapped there even once, it's considered mapped

Reply:
@akphi Tried the execution and does not work on the above cases. 
@pierredebelen confirmed that this is would work if you have the extends defined which we already addressed with https://github.com/finos/legend-studio/pull/822. 
@pierredebelen confirmed for the true `mappedness` of a subtype property we would need to go into the logic of reading and understanding operational class mappings. He does not want us to do this as it is complex. Instead we should only not show the property if we are confident it is not mapped. 
I opt to close this PR and leave it as showing all properties under subtype.